backend/api.py

```
@@ -1907,6 +1907,1047 @@ def process_reaper():
+        # Auto-initialize default mechanisms if database is empty
+        if not mechanisms:
+            print("📥 No mechanisms found - initializing default mechanisms with ETL...")
+            try:
+                from cancer_mechanism_loader import load_all_default_mechanisms
+                mechanism_ids = load_all_default_mechanisms(load_data=True)
+                if mechanism_ids:
+                    print(f"✅ Initialized {len(mechanism_ids)} mechanisms with data")
+                    mechanisms = get_all_mechanisms()
+                else:
+                    print("⚠️  Failed to initialize mechanisms")
+            except Exception as init_error:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)